community-operators/buildv2-operator/0.3.0/buildv2-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,251 @@
+          - get
+          - create
+        - apiGroups:
+          - apps
+          resourceNames:
+          - build-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - replicasets
+          - deployments
+          verbs:
+          - get
+        - apiGroups:
+          - build.dev
+          resources:
+          - '*'
+          - buildstrategies
+          - clusterbuildstrategies
+          - buildruns
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - tekton.dev
+          resources:
+          - tasks
+          - taskruns
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        serviceAccountName: build-operator
+      deployments:
+      - name: build-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: build-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: build-operator
+            spec:
+              containers:
+              - command:
+                - build-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: build-operator
+                image: quay.io/shipwright/shipwright-operator:v0.3.0
+                imagePullPolicy: Always
+                livenessProbe:
+                  exec:
+                    command:
+                    - stat
+                    - /tmp/operator-sdk-ready
+                  initialDelaySeconds: 5
+                  periodSeconds: 10
+                name: build-operator
+                readinessProbe:
+                  exec:
+                    command:
+                    - stat
+                    - /tmp/operator-sdk-ready
+                  initialDelaySeconds: 5
+                  periodSeconds: 10
+                resources: {}
+              serviceAccountName: build-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - API
+  - Cloud-native
+  - Build
+  maintainers:
+  - email: shbose@redhat.com
```

Comment:
Probably should use a mailing list address?

Reply:
Yes - update to shipwright-dev@lists.shipwright.io